package.json

```diff
@@ -67,13 +68,16 @@
     "async-retry": "^1.3.1",
     "bent": "^7.3.11",
     "ejs": "^3.1.5",
+    "odiff-bin": "^2.5.0",
     "pixelmatch": "^5.2.1",
     "pngjs": "^5.0.0",
+    "sharp": "^0.30.6",
```

Comment:
What does sharp do?

Reply:
It's for image manipulations - I use it to overlay the mask over the image and xor the masks. Should have added it to description but forgot - https://sharp.pixelplumbing.com/